solidity/dashboard/src/components/liquidity/RewardMultiplier.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { Skeleton } from "../skeletons"
+import CountUp from "react-countup"
+
+export const RewardMultiplier = ({
+  rewardMultiplier,
+  isFetching = false,
+  skeletonProps = { tag: "h2", shining: true, color: "grey-10" },
+  className = "",
+}) => {
+  return isFetching ? (
+    <Skeleton {...skeletonProps} />
+  ) : (
+    <h2 className={` ${className} liquidity__info-tile__title text-mint-100`}>
+      <CountUp
+        end={rewardMultiplier}
+        preserveValue
+        decimals={1}
+        duration={1}
+        suffix={"x"}
+      />
+    </h2>
+  )
+}
+
+RewardMultiplier.TooltipContent = () => {
+  // TODO: content of the tooltip
+  return <>Reward multiplier tooltip</>
```

Comment:
I realize that we don't have wireframes for this but maybe: `Deposit liquidity tokens for 30 days to achieve a 3x reward multiplier.` + something about how we calculate it? I mean we take into account all stakes. What do you think?